creator-node/src/config.js

```diff
@@ -661,12 +663,14 @@ const config = convict({
  * So if registryAddress or ownerWallet env variables are defined, they take precendence.
  */
 
+const pathTo = fileName => path.join(process.cwd(), fileName)
+
 // TODO(DM) - remove these defaults
 const defaultConfigExists = fs.existsSync('default-config.json')
 if (defaultConfigExists) config.loadFile('default-config.json')
 
-if (fs.existsSync('eth-contract-config.json')) {
-  let ethContractConfig = require('../eth-contract-config.json')
```

Comment:
was this a bug?

Reply:
it wasn't because the path to this file from the code stayed the same. now with the addition of the build/src/ folder these json files are actually one level higher than before so thats what this fix is for

Reply:
ah ok